src/wallet/coinselection.cpp

```diff
@@ -406,9 +406,15 @@ CAmount GenerateChangeTarget(CAmount payment_value, FastRandomContext& rng)
     }
 }
 
-void SelectionResult::ComputeAndSetWaste(CAmount change_cost)
+void SelectionResult::ComputeAndSetWaste(const CAmount min_change, const CAmount change_cost, const CAmount change_fee)
```

Comment:
In "wallet: use GetChange() when computing waste" (45ee9d3a737cc23babb9200c4d75d7dae4dc6700):

As suggested before, I'd prefer `min_viable_change` for this value here and below.

Reply:
done